src/test/scala/uk/gov/hmrc/ct/computations/CP281aSpec.scala

```diff
@@ -37,10 +37,10 @@ class CP281aSpec extends WordSpec with Matchers with MockitoSugar with BoxValida
   testCannotExistWhen("CP281a", CP281a.apply, testDetails = "CPQ17 false") {
     makeBoxRetriever(cpq17Value = false)
   }
-
-  testCannotExistWhen("CP281a", CP281a.apply, testDetails = "before loss reform") {
-    makeBoxRetriever(cp2Value = CPQ17.lossReform2017)
-  }
+//
+//  testCannotExistWhen("CP281a", CP281a.apply, testDetails = "before loss reform") {
+//    makeBoxRetriever(cp2Value = CPQ17.lossReform2017)
+//  }
```

Comment:
Another commented code. Usully it can be just deleted. Don't spend too much time in the investigation. Even when removed it will be still in the git history. @rutpatelhmrc

Reply:
Removed it as it wasn't required as the switch is already on for the loss reform 2017